### DISCUSSION TITLE
($250) Write command to turn this SVG into the closest possible rendering of this image

### DISCUSSION THREAD

**Discussion by Partytray** [ImageMagick/ImageMagick#6265] [Consulting]
I have various formats of the input (AI/EPS/whatever you need) and I need to get as close as possible to the output. Looking to make a move on this ASAP.

Input (this can be tweaked at design time if it requires to get the desired output):

![input](https://user-images.githubusercontent.com/97254326/233146123-62606c34-31bd-48cc-99b8-00a222bf90bd.svg)

Output:

![output-copy](https://user-images.githubusercontent.com/97254326/233124691-622f562a-84f8-4734-9991-2126ff78820c.jpg)

Of note: edges, depth.

Please inquire at jonlockdown[at]googlesmailservice.

**Comment by dlemstra** [comment]
Have you tried to do `magick -size 668x1504 input.svg output.png` ?

**Reply by Partytray** [reply]
Everything I do gets cluttered with antialiasing no matter what I try :(

**Comment by snibgo** [comment]
The supplied input is PNG, not SVG.

**Reply by Partytray** [reply]
Ah shoot, wrong file. Updated.

**Comment by snibgo** [comment]
You don't want anti-aliasing? Is that correct?

SVG is not highly prescriptive. Different agents can give different results for the same SVG. But this works when the IM delegate is Inkscape v1.2.

Using a text editor or sed or your favourite tool, change the SVG file so every "`<path`" becomes "`<path shape-rendering="crispEdges"`".

**Comment by snibgo** [comment]
Consider the top two of the four large shapes in the bottom half of the image. In the SVG, the top two shapes overlap in the Y-direction. In the required output, they do not overlap.

Hence the required output is not a simple rasterisation of the given SVG.

The colours are also different. As there are only 6 colours, this is fairly easily fixable.
